apps/site/src/pages/account/api-keys-panel/api-keys-list/api-key-table-row.tsx

```diff
@@ -0,0 +1,109 @@
+import { Box, TableCell, TableRow, Typography } from "@mui/material";
+
+import { CODE_FONT_FAMILY } from "../../../../theme/typography";
+import { useApiKeys } from "../api-keys-context";
+import { ApiKeyItemProps } from "../types";
+import { RowActions } from "./api-key-table-row/row-actions";
+import { formatDateForDisplay } from "./shared/format-date-for-display";
+import { NewIndicator } from "./shared/new-indicator";
+import { NewlyCreatedApiKeyCard } from "./shared/newly-created-api-key-card";
+
+const MaskedPublicId = ({ publicId }: { publicId: string }) => {
+  return (
+    <Typography
+      sx={{
+        fontFamily: CODE_FONT_FAMILY,
+        color: ({ palette }) => palette.gray[50],
+        fontSize: 14,
+        wordBreak: "break-word",
+      }}
+    >
+      b10ck5.
+      <Box component="span" sx={{ color: ({ palette }) => palette.gray[90] }}>
+        {publicId}
+      </Box>
+      .xxxxx
+    </Typography>
+  );
+};
+
+const DateText = ({ date }: { date?: Date | null }) => {
+  const formattedDates = formatDateForDisplay(date);
+
+  return (
+    <Typography>
+      {formattedDates.relative}
+      <Typography
```

Comment:
This is causing a DOM nesting error

```
xt-dev.js:23 Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
    at p
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Typography (webpack-internal:///../../node_modules/@mui/material/Typography/Typography.js:98:87)
    at p
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Typography (webpack-internal:///../../node_modules/@mui/material/Typography/Typography.js:98:87)
    at DateText (webpack-internal:///./src/pages/account/api-keys-panel/api-keys-list/api-key-table-row.tsx:61:11)
    at td
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at TableCell (webpack-internal:///../../node_modules/@mui/material/TableCell/TableCell.js:116:83)
    at tr
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at TableRow (webpack-internal:///../../node_modules/@mui/material/TableRow/TableRow.js:76:82)
    at ApiKeyTableRow (webpack-internal:///./src/pages/account/api-keys-panel/api-keys-list/api-key-table-row.tsx:89:19)
    at tbody
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at TableBody (webpack-internal:///../../node_modules/@mui/material/TableBody/TableBody.js:49:82)
    at table
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Table (webpack-internal:///../../node_modules/@mui/material/Table/Table.js:66:82)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at TableContainer (webpack-internal:///../../node_modules/@mui/material/TableContainer/TableContainer.js:44:82)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at ApiKeysList (webpack-internal:///./src/pages/account/api-keys-panel/api-keys-list.tsx:25:246)
    at div
    at PanelSection (webpack-internal:///./src/components/pages/account/panel-section.tsx:11:11)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Stack (webpack-internal:///../../node_modules/@mui/material/Stack/Stack.js:118:87)
    at ApiKeysPanel (webpack-internal:///./src/pages/account/api-keys-panel.tsx:40:82)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Paper (webpack-internal:///../../node_modules/@mui/material/Paper/Paper.js:77:83)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Container (webpack-internal:///../../node_modules/@mui/system/esm/Container/createContainer.js:116:19)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at PageContainer (webpack-internal:///./src/components/pages/dashboard/page-container.tsx:11:11)
    at Account (webpack-internal:///./src/pages/account/[[...slugs]].page.tsx:66:74)
    at AuthWallWrapper (webpack-internal:///./src/components/pages/auth-wall.tsx:15:11)
    at NextPageComponent
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at NavbarContainer (webpack-internal:///./src/components/navbar.tsx:683:11)
    at div
    at eval (webpack-internal:///../../node_modules/@emotion/react/dist/emotion-element-cbed451f.browser.esm.js:57:66)
    at Box (webpack-internal:///../../node_modules/@mui/system/esm/createBox.js:34:72)
    at PageLayout (webpack-internal:///./src/components/page-layout.tsx:28:11)
    at SnackbarProvider (webpack-internal:///../../node_modules/notistack/dist/notistack.esm.js:739:24)
    at InnerThemeProvider (webpack-internal:///../../node_modules/@mui/system/esm/ThemeProvider/ThemeProvider.js:21:70)
    at ThemeProvider (webpack-internal:///../../node_modules/@mui/private-theming/ThemeProvider/ThemeProvider.js:43:5)
    at ThemeProvider (webpack-internal:///../../node_modules/@mui/system/esm/ThemeProvider/ThemeProvider.js:40:5)
    at LazyMotion (webpack-internal:///../../node_modules/framer-motion/dist/es/components/LazyMotion/index.mjs:48:23)
    at MyApp (webpack-internal:///./src/pages/_app.page.tsx:85:11)
    at PathnameContextProviderAdapter (webpack-internal:///../../node_modules/next/dist/shared/lib/router/adapters.js:62:11)
    at ErrorBoundary (webpack-internal:///../../node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:301:63)
    at ReactDevOverlay (webpack-internal:///../../node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:850:919)
    at Container (webpack-internal:///../../node_modules/next/dist/client/index.js:62:1)
    at AppContainer (webpack-internal:///../../node_modules/next/dist/client/index.js:172:11)
    at Root (webpack-internal:///../../node_modules/next/dist/client/index.js:347:11)
w
```

Reply:
Fixed here https://github.com/blockprotocol/blockprotocol/pull/1257/commits/b4163e4dc020168c1141ca3793804b8285a15033